py/web/views/events.py

```diff
@@ -872,19 +872,25 @@ async def execute(self, m: Model):
 class EventUpdate(UpdateView):
     class Model(BaseModel):
         subject: constr(max_length=200)
-        message: str
+        message: Optional[str]
 
     async def check_permissions(self):
         await check_session(self.request, 'admin', 'host')
 
     async def execute(self, m: Model):
         event_id = await _check_event_permissions(self.request, check_upcoming=True)
+        payload_json = await self.request.json()
```

Comment:
doesn't really make sense to use the json payload when you have `Model`

Reply:
Ah yes. The view doesn't know what the possible id's that will receive messages. I haven't mastered pydantic yet and wasn't sure how to define a field with a wildcard or some other way to include all messages into `m`.